tsup.config.ts

```diff
@@ -16,10 +30,11 @@ const tsupConfig = defineTsupConfig({
   sourcemap: true,
   minify: true,
   clean: true,
-  dts: true,
   splitting: false,
   format: ["cjs", "esm"],
-  external: ["react", "@ark-ui/react", "react-icons", "next"],
+  // NB: `peerDeps`, among others, are excluded (marked external) by default
+  // see https://tsup.egoist.dev/#excluding-packages
+  external: ["@ark-ui/react", "react-icons", "next"],
```

Comment:
Verified bundle sizes were the exact same before and after removing `react`

Reply:
Ahhh does this change have something to do with the addition of `next`? Just out of curiosity.

Reply:
Nah, just an oversight on my part, `tsup` automatically excludes peer deps

Reply:
Gotcha!